install.sh

```diff
@@ -115,43 +119,45 @@ while [ "$done" == false ]; do
     "Restore Panel Backup (To remove some addon and restore your old panel.)"
     "Install More Buttons (Only 1.6.6 and 1.7.0)"
     "Install More Server Info (Only 1.6.6 and 1.7.0)"
+    "Install Server Router Icons (Only 1.6.6 and 1.7.0)"
```

Comment:
I forgot to mention this, but it works for `1.6.6` and `1.7.0`?

Reply:
Should work for any 1.x version